roles/edpm_extra_mounts/meta/argument_specs.yml

```diff
@@ -11,20 +11,19 @@ argument_specs:
           A list of additional mounts (e.g. for nfs) to be added to the edpm compute node.
           The list should be in the form of a list of dictionaries with the
           following keys:
-          - name: The name of the mount by default /var/lib/nova/instances
-          - path: The path to the mount
+          - path: The name of the mount by default /var/lib/nova/instances
+          - src: The path to the mount
```

Comment:
Both the descriptions do not seem correct. `src` is `Device (or NFS volume, or something else) to be mounted on path.`

Reply:
Thanks! Done